src/bin/submitpackage/main.rs

```diff
@@ -0,0 +1,78 @@
+use base64::{engine::general_purpose, Engine as _};
+use bitcoin::network::Magic;
+use clap::{Parser, ValueEnum};
+use hex_string::HexString;
+use nostr::prelude::*;
+use nostr::Keys;
+use nostr_sdk::Client;
+
+#[derive(Parser)]
+#[command()]
+struct Args {
+    #[arg(short, value_enum)]
+    network: Network,
+
+    #[arg(short)]
+    relay: String,
```

Comment:
Done. Can't use num_args here, because otherwise it gets confused as to what is a relay and what a tx in for example the following cmd line:

`submitpackage -r host1 host2 904...589495  2894...1727`